add CHANGELOG entries for #234 and #235 (#236)
hashicorp/terraform-provider-nomad: CHANGELOG.md

## 1.4.16 (Unreleased)

BUG FIXES:
* data source/nomad_plugin: wait for the correct amount of expected controllers ([#234](https://github.com/hashicorp/terraform-provider-nomad/pull/234))

> * data source/nomad_plugin: wait for healthy nodes as well ([#235](https://github.com/hashicorp/terraform-provider-nomad/pull/235))

Reviewer: ```suggestion
* data source/nomad_plugin: wait for the correct amount of healthy nodes ([#235](https://github.com/hashicorp/terraform-provider-nomad/pull/235))
```
